src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -167,7 +181,17 @@ void MusicBrainzRecordingsTask::doNetworkReplyFinished(
 
     // Continue with next recording id
     DEBUG_ASSERT(!m_queuedRecordingIds.isEmpty());
-    slotStart(m_parentTimeoutMillis);
+    auto timerSinceLastRequest = m_measurementTimer.elapsed(true);
```

Comment:
What is all this unneeded extra logic for? It has become obsolete.

Who is actually reviewing this code???

Reply:
> It has become obsolete.

Why, please explain.